meilisearch-http/src/routes/mod.rs

```diff
@@ -266,108 +270,6 @@ struct KeysResponse {
     public: Option<String>,
 }
 
-pub async fn list_keys(meilisearch: GuardedData<Admin, ApiKeys>) -> HttpResponse {
-    let api_keys = (*meilisearch).clone();
-    HttpResponse::Ok().json(&KeysResponse {
-        private: api_keys.private,
-        public: api_keys.public,
-    })
-}
-
 pub async fn get_health() -> Result<HttpResponse, ResponseError> {
     Ok(HttpResponse::Ok().json(serde_json::json!({ "status": "available" })))
 }
-
-#[cfg(test)]
```

Comment:
I am quite sad to see that go. It statically checked that the routes were defined with the correct authentication, and this doesn't seem to be checked anywhere else...

Reply:
I can make mistakes but I think the [authorization test file](https://github.com/meilisearch/MeiliSearch/pull/1890/files#diff-55ad1f1508692d9ba5458690ea22eea9c10c41f5cea8aa360341e8f3a7891a31) cover it. What do you think? 🤔